src/EventSubscriber/HttpCacheResponseSubscriber.php

```diff
@@ -40,7 +40,7 @@ public static function getSubscribedEvents()
     public function configureCache(FilterResponseEvent $event)
     {
         $view = $event->getRequest()->attributes->get('view');
-        if (!$view instanceof CachableView || !$view->isCacheEnabled()) {
+        if ($view !== null && ( !$view instanceof CachableView || !$view->isCacheEnabled() )) {
```

Comment:
why this change? We don't want to set ttl on non ContentView response automatically. People can set their own s-maxage depending on what logic they have and what they need.

Reply:
Rest response are not `views`, we would 1return` then, the response would not be taggued and would be http cached